src/content/post/20161122_drupal_application_bottlenecks_avoid_with_profiling.md

```diff
@@ -0,0 +1,160 @@
++++
+date = "2016-11-22T11:02:30Z"
+draft = false
+title = "Avoid bottlenecks on a Drupal web application and how identify it with profiling"
+description = "How identify bottleneck of your web application and avoid that can appear in the future"
+tags        = [ "profiling", "drupal", "performance", "web application"  ]
+topics      = [ "Profiling", "Drupal", "Performance" ]
+author = "Vinz"
++++
+
+### TL;DR
```

Comment:
`## Introduction` it's ok for me. I've used TL;DR based upon this paragraph in the wikipedia page `The tl;dr label is often used to point out grossly excessive verbosity, or to introduce a short summation of a longer piece.` but is not important